platform-sdk/swirlds-common/src/testFixtures/java/com/swirlds/common/test/fixtures/merkle/util/PairedStreams.java

```diff
@@ -80,29 +80,39 @@ public MerkleDataInputStream getLearnerInput() {
     }
 
     @Override
-    public void close() throws IOException {
-        teacherOutput.close();
-        teacherInput.close();
-        learnerOutput.close();
-        learnerInput.close();
-
-        teacherOutputBuffer.close();
-        teacherInputBuffer.close();
-        learnerOutputBuffer.close();
-        learnerInputBuffer.close();
-
-        server.close();
-        teacherSocket.close();
-        learnerSocket.close();
+    public void close() {
+        try {
+            teacherOutput.close();
+            teacherInput.close();
+            learnerOutput.close();
+            learnerInput.close();
+
+            teacherOutputBuffer.close();
+            teacherInputBuffer.close();
+            learnerOutputBuffer.close();
+            learnerInputBuffer.close();
+
+            server.close();
+            teacherSocket.close();
+            learnerSocket.close();
+        } catch (IOException e) {
+            // this is the test code, and we don't want the test to fail because of a close error
```

Comment:
We actually do want it to fail if the stream was opened and the close() fails for some reason. It's better to catch this in a test than in production. But okay.

Reply:
This `PairedStreams` class is only used in tests, not in production

Reply:
The `close()` method in the stream implementations runs in production, and this fix swallows an exception thrown by that production method. So my point still stands.